podcasts/SleepTimerManager.swift

```diff
@@ -39,6 +49,16 @@ class SleepTimerManager {
         }
     }
 
+    func restartSleepTimer() {
+        if let setting = Settings.sleepTimerLastSetting {
+            if let duration = setting.duration {
+                PlaybackManager.shared.setSleepTimerInterval(duration)
+                Analytics.shared.track(.playerSleepTimerRestarted, properties: ["time": duration, "reason": "device_shake"])
```

Comment:
@mebarbosa I added a `reason: device_shake` in this event in case we want to differentiate from the rest in the future.

Reply:
I will update Android to match with it 👌🏽